cal/cal/static/css/calendaranalytics.css

```diff
@@ -10,22 +10,41 @@ h1, h2, h3, h4, h5, h6, a, p {
   color: #000; }
 
 .jumbotron h1, .jumbotron .h1 {
-  color: #000; }
+  color: #000;
```

Comment:
Command: sass --watch .
1) In our project root folder, run the command. This sets sass to watch for any changes in .scss files
2) modify the scss files you copied stuff over into, and save. This is to trigger a recompiling of the css files.

Let me know if you need more help. Here's some sass docs too: http://sass-lang.com/guide